astroquery/esa/xmm_newton/core.py

```diff
@@ -101,13 +103,21 @@ def download_data(self, observation_id, filename=None, verbose=False,
         if verbose:
             log.info(link)
 
-        response = self._request('GET', link, save=True, cache=True)
+        response = self._request('GET', link, save=False, cache=True)
```

Comment:
can this be achieved with a `'HEAD'` request?

Reply:
What exactly does that do?

Reply:
HTTP HEAD requests: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD are a way to just get the headers, not the data - it is presumably quite a bit cheaper than a full `GET` request.

Reply:
Ah ok, If we only get the header to get the headers we would make a` 'HEAD'`, but we want to download it not only to get his name.We can implement a GET and a HEAD for get the filename, but the GET method already returns everything we need

Reply:
ah, of course, you're right - you're doing everything in one request.  The approach you took here is totally fine.

Reply:
But I think you're right, I'm gonna change a few things